frontend/public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
plik do wywalenia

Reply:
nie jestem pewien, bo w niim jest chyba wszystko renderowane - zobacz do pliku src/index.tsx

Reply:
Moze i tak ale da sie to zrobic bez tego html'a. Nie ma sensu trzymac kolejnego pliku do maintainowania imo

Reply:
na razie wyczyściłem trochę tego htmla, w najblizszym czasie jak ogarne jak to zrobic bez niego to go wywalę